lib/src/models/user.dart

```diff
@@ -67,20 +67,20 @@ class User {
     user.cotter = cotter;
     // if this fails, then the user is not logged-in
     // or the refresh token failed to refresh
-    await Token.getIDToken(user.issuer);
+    await Token.getIDToken(user.issuer!);
```

Comment:
Does your API ever return null for the issuer?

Reply:
It shouldn't return null for issuer, so this should be good

Reply:
In that case, you could update the User class def to have a non-nullable string instead and remove all the `String?` -> `String` casts